src/coreclr/jit/gentree.cpp

```diff
@@ -6621,42 +6621,38 @@ bool GenTree::OperIsImplicitIndir() const
 //    Should not be used on calls given that we can say nothing precise about
 //    those.
 //
-ExceptionSetFlags GenTree::OperExceptions(Compiler* comp)
+ExceptionSetFlags GenTree::GetExceptionSetFlags(Compiler* comp)
```

Comment:
I don't think you should rename this. `OperExceptions` follows the convention that this is an operation that operates on the root-level node only and does not take child node into account. It is a convention used in many other places (`OperRequiresCallFlag`, `OperRequiresAsgFlag`, `OperMayThrow` etc.).

Reply:
Sure, I can revert it. The name `OperExceptions` is a little confusing though - but happy to follow the conventions and keep the original name.